charts/nethermind/templates/_cmd.tpl

```diff
@@ -26,6 +26,10 @@
   --JsonRpc.Port={{ include "nethermind.httpPort" . }}
   --Init.WebSocketsEnabled=true
   --JsonRpc.WebSocketsPort={{ include "nethermind.httpPort" . }}
+  --JsonRpc.JwtSecretFile=/data/jwt.hex
+  --JsonRpc.EngineHost=0.0.0.0
+  --JsonRpc.EnginePort={{ include "nethermind.authPort" . }}
+  --JsonRpc.EnabledModules=["net","eth","subscribe","engine","web3","client"]
```

Comment:
As mentioned already on other EL client charts: Let'se remove `--JsonRpc.EnabledModules` from here and use extraArgs for deployment specific config.

Reply:
Should I continue passing these flags on our mainnet nodes?

Reply:
Yeah, for the mainnet nodes you can pass that on as extraArgs. But just ` --JsonRpc.EnabledModules=["net","eth","subscribe","engine","web3","client"]`. The rest you can keep within the chart as they're kinda required for running a node.